style.css

```diff
@@ -0,0 +1,70 @@
+body {
+  display: flex;
+  justify-content: center;
+}
+
+header {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 35%;
+  min-height: 100vh;
+  align-content: space-between;
+  position: absolute;
+  font-family: Arial, Helvetica, sans-serif;
+  background-image: url("images/header-shapes\ mobile.png");
```

Comment:
As required the previous reviewer you have to correct the background image:

use this code to cover the section and remove the repeat

```
background-repeat: no-repeat;
background-size: cover;
```